### ISSUE TITLE
[Accessibility] [Supporting the platform -Tags]: Focus indicator is not visible on the “Improve this page” link in high-contrast mode. [Sev3]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#43]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Pre-requisites:**
Settings > Accessibility > Contrast themes > Select ‘Aquatic’ and 'Dessert' modes.

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now navigate to ‘Tags’ link and activate it.
4.	Now, enable high-contrast mode in settings and check whether the focus indicator is visible on the ‘Improve this page’ link or not.

**Actual:**
Focus indicator is not visible on the ‘Improve this page’ link in ‘tags ‘page in high-contrast mode.

**Expected:**
Focus indicator should be visible on the ‘Improve this page’ link in ‘tags’ page in high-contrast mode.

**Note:**
Issue is observed for the ‘Improve this page’ link in ‘GitHub’, ‘Help’, ‘@return’, ‘Submitting a PR’ pages.

**User Impact:**
When focus indicator is not visible on the interactive elements in high-contrast mode, customers who rely on that feature are blocked from information and functionality of the full user experience within the site/app.

**Guideline Reference:**
EN 301 549 V3.2.1 - 11.6.2.0 No disruption of accessibility features

https://user-images.githubusercontent.com/88436727/151991379-3107eb6a-6103-47c5-bcb3-adf0824d507c.mp4
